### PR TITLE
Create ui-c8y-1020-23-1-error-appears-for-checking-active-filters-on-subtenants-6904-graft-release.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-59337](https://cumulocity.atlassian.net/browse/MTM-59337)
Original PR: [6981](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6981)

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1020-23-1-error-appears-for-checking-active-filters-on-subtenants-6904-graft-release.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: error appears for checking active filters on subtenants (#6904) [GRAFT][release/cd] (#6981)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-23-1-error-appears-for-checking-active-filters-on-subtenants-6904-graft-release.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Fix error when checking active filters on subtenants
```

File: content/change-logs/application-enablement/ui-c8y-1020-23-1-error-appears-for-checking-active-filters-on-subtenants-6904-graft-release.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-59337
+version: 1020.23.1
+---
+error appears for checking active filters on subtenants (#6904) [GRAFT][release/cd] (#6981)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-23-1-error-appears-for-checking-active-filters-on-subtenants-6904-graft-release.md:17]: The Claude AI suggested to use the following body:
```suggestion
In some situations, an error occurred when checking active filters on subtenants, preventing users from properly viewing and managing the filters. This issue has been resolved. Subtenants can now reliably check their active filters without encountering an error. This change ensures that the filter management functionality works as expected for all subtenants.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Comment by BeateRixen** [comment]
@pmich-sag Please review and approve

File: content/change-logs/application-enablement/ui-c8y-1020-23-1-error-appears-for-checking-active-filters-on-subtenants-6904-graft-release.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
```
Comment (pmich-sag) [inline content/change-logs/application-enablement/ui-c8y-1020-23-1-error-appears-for-checking-active-filters-on-subtenants-6904-graft-release.md:17]:
```suggestion
In some situations, an error occurred when checking active filters on subtenants, preventing users from properly viewing and managing the filters. This issue has been resolved. Subtenants can now reliably check their active filters without encountering an error. This change ensures that the filtering functionality works as expected for all subtenants.
```

File: content/change-logs/application-enablement/ui-c8y-1020-23-1-error-appears-for-checking-active-filters-on-subtenants-6904-graft-release.md (hunk 2 above)
Comment (pmich-sag) [reply content/change-logs/application-enablement/ui-c8y-1020-23-1-error-appears-for-checking-active-filters-on-subtenants-6904-graft-release.md]: Resolved.

File: content/change-logs/application-enablement/ui-c8y-1020-23-1-error-appears-for-checking-active-filters-on-subtenants-6904-graft-release.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-59337
+version: 1020.23.1
+---
+In some situations, an error occurred when checking active filters on subtenants, preventing users from properly viewing and managing the filters. This issue has been resolved. Subtenants can now reliably check their active filters without encountering an error. This change ensures that the filtering functionality works as expected for all subtenants.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-23-1-error-appears-for-checking-active-filters-on-subtenants-6904-graft-release.md:17]:
```suggestion
In some situations an error occurred when checking active filters on subtenants, preventing users from properly viewing and managing the filters. This issue has been resolved. Subtenants can now reliably check their active filters without encountering an error. 
```

File: content/change-logs/application-enablement/ui-c8y-1020-23-1-error-appears-for-checking-active-filters-on-subtenants-6904-graft-release.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Fix error when checking active filters on subtenants
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-23-1-error-appears-for-checking-active-filters-on-subtenants-6904-graft-release.md:3]:
```suggestion
title: Subtenants can now reliably check their active filters
```

File: content/change-logs/application-enablement/ui-c8y-1020-23-1-error-appears-for-checking-active-filters-on-subtenants-6904-graft-release.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Subtenants can now reliably check their active filters
+product_area: Application enablement & solutions
+change_type:
+  - value: change-VSkj2iV9m
+    label: Fix
+component:
+  - value: component-YbYJ3gLU_
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-23-1-error-appears-for-checking-active-filters-on-subtenants-6904-graft-release.md:9]:
```suggestion
  - value: component-0UgqXH1Ys 
```

File: content/change-logs/application-enablement/ui-c8y-1020-23-1-error-appears-for-checking-active-filters-on-subtenants-6904-graft-release.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Subtenants can now reliably check their active filters
+product_area: Application enablement & solutions
+change_type:
+  - value: change-VSkj2iV9m
+    label: Fix
+component:
+  - value: component-0UgqXH1Ys 
+    label: Web SDK
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-23-1-error-appears-for-checking-active-filters-on-subtenants-6904-graft-release.md:10]:
```suggestion
    label: Administration
```

File: content/change-logs/application-enablement/ui-c8y-1020-23-1-error-appears-for-checking-active-filters-on-subtenants-6904-graft-release.md [hunk 8]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Subtenants can now reliably check their active filters
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-23-1-error-appears-for-checking-active-filters-on-subtenants-6904-graft-release.md:3]:
```suggestion
title: Users can now reliably check active filters for subtenants
```

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**